packages/strategy-tests/src/lib.rs

```diff
@@ -418,7 +418,9 @@ impl Strategy {
         let (identities, mut state_transitions): (Vec<Identity>, Vec<StateTransition>) =
             identity_state_transitions.into_iter().unzip();
 
-        // Do we also need to add identities to the identity_nonce_counter?
+        for identity in &identities {
+            identity_nonce_counter.insert(identity.id(), 1);
+        }
```

Comment:
I think this means we would start things at 2.

Reply:
Fixed (removed it because it wasn't necessary)